### ISSUE TITLE
Open state of dropdown always shows first item as selected

### ISSUE THREAD

**Issue by vpeti** [vitmalina/w2ui#930]
I have a dropdown where I noticed some weird behavior. This example works the same: http://w2ui.com/web/docs/form/fields-list

The problem is that after you select an item from the list and open the list again, always the first item gets selected.

After some investigation I think I found the relevant lines in the mresize method. When I was debugging it, the options.index always stayed at 0, explaining the behavior:

```javascript
$('#w2ui-overlay'+ name +' tr.w2ui-selected').removeClass('w2ui-selected');
var cur        = $('#w2ui-overlay'+ name +' tr[index='+ options.index +']');
var scrTop    = $('#w2ui-overlay'+ name +' div.menu').scrollTop();
cur.addClass('w2ui-selected');
```

**Comment by vitmalina** [comment]
Yes, it appears to be a bug. I think, I might have a fix.

**Comment by mrg2001** [comment]
I have just checked, it is fixed in 1.5 branch. See in repo /test/fields.html as an example
